docs/DatabaseCatalogAPI.yaml

```
@@ -972,6 +1010,18 @@ components:
           type: string
           format: url
           example: https://www.ladottransit.com/dla.html
+        license_id:
+          description: Id of the feed license that can be used to query the license endpoint.
+          type: string
+          example: 0BSD
+        license_is_spdx:
+          description: true if the license is SPDX. false if not.
+          type: boolean
+          example: true
+        license_notes:
+          description: TBD
```

This is a generic text notes field. We can document it with:

```suggestion
          description: Notes associated with the feed license.
```